app/assets/javascripts/channels/room.coffee

```diff
@@ -0,0 +1,33 @@
+App.room = App.cable.subscriptions.create "RoomChannel",
+  connected: ->
+    # Called when the subscription is ready for use on the server
+
+  disconnected: ->
+    # Called when the subscription has been terminated by the server
+
+  received: (data) ->
+    unless data.content.blank?
+
+      html = '<div class="message">'
+      html += '<div class = "msg-bubble">'
+      html += '<div class = "msg-info">'
+      html += '<div class="message-user"><strong>'
+      html += data.first_name
+      html += ':</strong></div>'
+      html += '<div class="message-time">'
+      html += new Date().toLocaleString()
+      html += '</div>'
+      html += '</div>'
+      html += '<div class="message-content"><p>'
+      html += data.content 
```

Comment:
I really like this format, It's super clear and easy to read.

Reply:
Indeed. Changed it to this as it was hard to work on the one from the example we used